DB/data/on_startup.sql

```diff
@@ -1,16 +1,16 @@
 delete from domain where name ~* 'Digital products';
 delete from domain where name ~* 'Recruitment';
 
-insert into domain(id, name, ordering) values
-  (1, 'Strategy and Policy',                          1),
-  (2, 'Change, Training and Transformation',          7),
-  (3, 'User research and Design',                     2),
-  (4, 'Agile delivery and Governance',                3),
-  (6, 'Software engineering and Development',         4),
-  (7, 'Content and Publishing',                       6),
-  (8, 'Cyber security',                               9),
-  (9, 'Marketing, Communications and Engagement',     8),
-  (10, 'Support and Operations',                      5),
-  (11, 'Data science',                                10),
-  (13, 'Emerging technologies',                         11)
+insert into domain(id, name, ordering, price_minimum, price_maximum) values
```

Comment:
does `on_startup` happen everytime the app starts?  would this not be better just as a one-off script?